doc/sphinx/Readme.rst

```diff
@@ -5,7 +5,8 @@ How to get started with Sphinx
 
    .. code-block:: bash
 
-       pip3 install --user --upgrade 'sphinx>=2.3.0,!=3.0.0' 'sphinxcontrib-bibtex>=0.3.5'
+       python3 -m pip install -c requirements.txt \
+         sphinx sphinx-toggleprompt sphinxcontrib-bibtex numpydoc pybtex
```

Comment:
What do we need `pybtex` for?

Reply:
https://github.com/espressomd/espresso/blob/adf636d9916a19485604c376d925d7849fba3076/doc/sphinx/conf.py.in#L279-L282